src/services/decoders/ClpIrDecoder/index.ts

```diff
@@ -188,6 +172,26 @@ class ClpIrDecoder implements Decoder {
 
         return results;
     }
+
+    /**
+     * Formats all log messages by prepending a dayjs formatted timestamp to beginning or each
+     * message
+     *
+     * @param logEvents
+     * @return The formatted log events.
+     */
+    #formatResults (logEvents: DecodeResult[]): Nullable<DecodeResult[]> {
```

Comment:
how about renaming it as `formatUnstructuredResults` since it is only used for unstructured logs?